src/screens/DetailedViewScreen.tsx

```diff
@@ -106,29 +108,43 @@ const DetailedViewScreen: React.FC<PlaceProps> = ({ setPage, placeID }: PlacePro
 							destination={placeDetails.name}
 							destination_place_id={placeID}
 							formatted_address={placeDetails.formatted_address}
+							callback={setIsOpenInMapsTapped}
 						>
-							<Button
-								style={{ ...styles.button, display: "flex" }}
-								onPress={() => {
-									return;
+							<View
+								style={{
+									...styles.button,
+									display: "flex",
+									flexDirection: "row",
+									alignItems: "center",
+									justifyContent: "center",
+									paddingHorizontal: 15,
+									paddingVertical: 10,
+									backgroundColor: isOpenInMapsTapped ? "black" : "white",
```

Comment:
can we use the color constants from `colors.ts`?

Reply:
mixed feelings... i feel like it's way more clear to do it like it is than it is to do `backgroundColor: isOpenInMapsTapped ? TEXT_COLOR : PRIMARY_COLOR`

but... i'll change it for now

Reply:
yeah I get it... but just in case we ever change what color theme the app uses I think this would save some headache